### PR TITLE
fix: Leprechaun rebalance

### PR DESCRIPTION
**Pull request by Golbolco**
Rebalanced how many clovers are given out at night. Big thanks to Sand for helping code and testing!

### CODE REVIEW

File: Games/types/Mafia/roles/cards/WinByStealingClovers.js [hunk 1]
```diff
@@ -26,14 +26,12 @@ module.exports = class WinByStealingClovers extends Card {
...
-
-        let numCloversToSpawn = Math.round(
-          Math2.lerp(this.data.cloverTarget, this.game.players.length, 0.8)
-        );
-
         let eligiblePlayers = this.game.players.filter(
           (p) => p.role.name !== "Leprechaun"
         );
+
+        let numCloversToSpawn = this.data.cloverTarget + (this.game.players.length - eligiblePlayers.length);
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/WinByStealingClovers.js:33]: this is just 3 + numLeprechaun

File: Games/types/Mafia/roles/cards/StealAllItemsAndClovers.js [hunk 2]
```diff
@@ -2,45 +2,48 @@ const Card = require("../../Card");
...
+            "Steal From": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive", "dead"], exclude: ["self"] },
+                action: {
+                    labels: ["stealItem", "kill"],
+                    priority: PRIORITY_ITEM_TAKER_DEFAULT,
+                    run: function() {
+                        let stealItem = false;
+                        let killActor = false;
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/StealAllItemsAndClovers.js:19]: stealItem has no function

File: Games/types/Mafia/roles/cards/WinByStealingClovers.js (hunk 1 above)
Comment (Golbolco) [reply Games/types/Mafia/roles/cards/WinByStealingClovers.js:33]: For now it's identical, but if more roles were filtered from eligiblePlayers in the future, then the way it's written now will make this file easier to expand with only another line of code per role.

File: Games/types/Mafia/roles/cards/StealAllItemsAndClovers.js (hunk 2 above)
Comment (Golbolco) [reply Games/types/Mafia/roles/cards/StealAllItemsAndClovers.js:18]: removed

File: Games/types/Mafia/roles/cards/WinByStealingClovers.js (hunk 1 above)
Comment (psammite) [reply Games/types/Mafia/roles/cards/WinByStealingClovers.js:33]: don't we want numCloversToSpawn to always be 3 + numLeprechaun? filtering more roles from eligiblePlayers will mess this up

**Review by pikulet — APPROVED** [review]

**Pull request merged**